packages/frontend/src/common/components/TimePicker.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { FlexBox } from "./FlexBox";
+import { TextInput } from "./TextInput";
+import IncrementTimerButton from "../../retro/components/buttons/IncrementTimerButton";
+interface TimePickerProps {
+  minutes: string;
+  seconds: string;
+  isEditable: boolean;
+  isMinutesError: boolean;
+  isSecondsError: boolean;
+  onSecondsChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onMinutesChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onEnter: () => void;
+  onTimerIncrement: (increment: number) => void;
+}
+export function TimePicker({
+  minutes,
+  seconds,
+  isEditable,
+  isMinutesError,
+  isSecondsError,
+  onSecondsChange,
+  onMinutesChange,
+  onEnter,
+  onTimerIncrement,
+}: TimePickerProps) {
+  return (
+    <>
+      <FlexBox gap={2}>
+        <TextInput
+          value={minutes}
+          onChange={onMinutesChange}
+          onSubmit={onEnter}
+          autoFocus
+          required
+          disabled={!isEditable}
+          type="tel"
+          error={isMinutesError}
+          label="Minutes"
+        />
+        <TextInput
+          value={seconds}
+          onChange={onSecondsChange}
+          onSubmit={onEnter}
+          required
+          type="tel"
+          disabled={!isEditable}
+          error={isSecondsError}
+          label="Seconds"
+        />
+      </FlexBox>
+      <FlexBox>
+        <IncrementTimerButton
+          label={"+1 Minutes"}
```

Comment:
We can determine the label inside this component, since it only depends on the incrementAmount.